### ISSUE TITLE
Error when other events are used on the page

### ISSUE THREAD

**Issue by timukasr** [web-eid/web-eid.js#8]
Library listens for all the events of the page and logs following error when they do not contain `action` field in data:

```
web-eid-1.0.0.js:376 Uncaught TypeError: Cannot read property 'replace' of undefined
    at WebExtensionService.getInitialAction (web-eid-1.0.0.js:376)
    at WebExtensionService.receive (web-eid-1.0.0.js:306)
    at web-eid-1.0.0.js:300
```

You can test it by writing `postMessage('test')` in https://web-eid.eu/ Javascript console.

Would be better to ignore other events.

**Comment by mrts** [comment]
Thank you, makes sense! We will take care of this when time allows.

**Comment by borm** [comment]
please fix it as soon as possible

**Comment by mrts** [comment]
We will - @taneltm, can you comment on the likely time of delivery of the fix?

**Comment by taneltm** [comment]
Latest on Friday. Another project took priority for a short while.

**Comment by mrts** [comment]
@timukasr, @borm the fix is now available from https://github.com/web-eid/web-eid.js/pull/10/checks > _Artifacts_. Can you please verify that this solves the problem for you?

Thanks in advance!

**Comment by timukasr** [comment]
Yes, this solves the problem for me. Thank you.

**Comment by borm** [comment]
how i can get new version? 
```
"dependencies": {
   ...
    "@web-eid/web-eid-library": "latest",
```

```
rm -rf node_modules
yarn install
```

but, problem still exists 

```
Couldn't find any versions for "@web-eid/web-eid-library" that matches "1.0.1"
? Please choose a version of "@web-eid/web-eid-library" from this list: (Use arrow keys)
❯ 1.0.0 
  0.9.0 
  0.1.3 
  0.1.2 
  0.1.1 
  0.1.0 
  0.0.7 
```

did u published it? @mrts 
https://gitlab.com/api/v4/packages/npm/@web-eid/web-eid-library
```
{"latest":"1.0.0"}
```

**Comment by mrts** [comment]
> how i can get new version?

Haven't published it yet. We have a semi-formal internal publishing schedule, I'll see if I can get this published soon outside of the usual release train. Will keep you updated.

You can download the artifacts and include the module manually meanwhile for testing.

**Comment by mrts** [comment]
@borm, I published v1.0.1. Hope everything works well now, happy hacking!

**Comment by borm** [comment]
@mrts thank you so much, everything works well
happy ;)
